### ISSUE TITLE
Is any chance to make the similar String picker not only time?

### ISSUE THREAD

**Issue by ADevelopStudio** [dariopellegrini/DPTimePicker#2]

**Comment by dariopellegrini** [comment]
Yes, actually time number values are string arrays inside DPTimePicker.swift file and they're built using cycles in insertInView func. You can modify those cycles and put inside them strings array. For me it's a bit difficult to update right now, but for sure I'll do asap.
